modules/Bio/EnsEMBL/Analysis/Hive/Config/long_read.pm

```diff
@@ -0,0 +1,589 @@
+
+    use_threads => 3,
+
+    # Please assign some or all columns from the summary file to the
+    # some or all of the following categories.  Multiple values can be
+    # separted with commas. ID, SM, DS, CN, is_paired, filename, read_length, is_13plus,
+    # is_mate_1 are required. If pairing_regex can work for you, set is_mate_1 to -1.
+    # You can use any other tag specified in the SAM specification:
+    # http://samtools.github.io/hts-specs/SAMv1.pdf
+
+    ####################################################################
+    # This is just an example based on the file snippet shown below.  It
+    # will vary depending on how your data looks.
+    ####################################################################
+    long_read_columns => [ 'sample', 'filename' ],
+
+    # !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+    # No option below this mark should be modified
+    # !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+    ########################
+    # db info
+    ########################
+    long_read_initial_db => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_lrinitial_' . $self->o('release_number'),
+      -host   => $self->o('long_read_initial_db_server'),
+      -port   => $self->o('long_read_initial_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    long_read_collapse_db => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_lrcollapse_' . $self->o('release_number'),
+      -host   => $self->o('long_read_collapse_db_server'),
+      -port   => $self->o('long_read_collapse_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+    # TODO
+    # Delete or uncomment, referenced in commented out analyses of this pipeline.
+    # long_read_blast_db => {
+    #   -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_lrblast_' . $self->o('release_number'),
+    #   -host   => $self->o('long_read_blast_db_server'),
+    #   -port   => $self->o('long_read_blast_db_port'),
+    #   -user   => $self->o('user'),
+    #   -pass   => $self->o('password'),
+    #   -driver => $self->o('hive_driver'),
+    # },
+
+    long_read_final_db => {
+      -dbname => $self->o('dbowner') . '_' . $self->o('production_name') . '_lrfinal_' . $self->o('release_number'),
+      -host   => $self->o('long_read_final_db_server'),
+      -port   => $self->o('long_read_final_db_port'),
+      -user   => $self->o('user'),
+      -pass   => $self->o('password'),
+      -driver => $self->o('hive_driver'),
+    },
+
+  };
+}
+
+sub pipeline_create_commands {
+  my ($self) = @_;
+
+  return [
+
+    # inheriting database and hive tables' creation
+    @{ $self->SUPER::pipeline_create_commands },
+
+    'mkdir -p ' . $self->o('long_read_fastq_dir'),
+  ];
+}
+
```

Comment:
I would be tempted to remove this as the directory is created during the pipeline. But there is an argument to have it here instead of in the pipeline.

Reply:
Keeping it for now.